### DISCUSSION TITLE
Is Olive development still alive?

### DISCUSSION THREAD

**Discussion by therealrobster** [olive-editor/olive#2127] [General]
Hi there,

I was very much into olive a few years ago but paused its use whilst 0.2 happened.  I just dropped in after a long break and have not found any movement for a while.

Just checking in if development has ceased?

Thanks so much...

**Comment by itsmattkc** [comment]
Yes it is still active. Development has slowed a little for the past few weeks due to work obligations, but will be restored to normal activity very soon.

**Comment by therealrobster** [comment]
ahhh ok, no stress at all and no expectation from me.  I was just interested.  I didn't see any code acivity in the github codebase so just wondered.  Great to hear, I am excited for when this is released for sure.  However, I just saw the nightly is Version: af76fbf0 (Dec 9 2022 5:25:31 AM) so I should have paid attention.  Grabbing now.  Thanks again!

**Comment by Suk-RAM** [comment]
I think nobody should complain about the speed of Olive development because Olive is a project of volunteers.
But when Olive's website shows for month or even years that Olive 0.2 is coming soon then this is misleading.
